src/scaffolding/schema-builder.ts

```diff
@@ -70,7 +69,6 @@ export class SchemaBuilder {
       schema.model.toHuman() === JSON.stringify(this.values.model) &&
       schema.model_type.type === this.values.modelType &&
       schema.payload_location.type === this.values.payloadLocation &&
-      new Set(schema.settings.toArray().map((s) => s.toString())) === new Set(this.values.settings ? this.values.settings : []) &&
       JSON.stringify(schema.settings.toArray()) === JSON.stringify(this.values.settings ? [this.values.settings] : [])
```

Comment:
The JSON line does this and the set comparison will always fail as they test object equality not content equality.